main/test/flix/fuzzers/FuzzerSuite.scala

```diff
@@ -19,5 +19,10 @@ import org.scalatest.Suites
 
 class FuzzerSuite extends Suites(
   new FuzzPrefixes,
-  new FuzzLines
+  new FuzzDeleteLines,
+  new FuzzDuplicateLines,
+  // Swap lines takes a _long_ time to run.
+  // In a file with just 100 lines, there is (100 * 99) / 2 unique non-equal swaps.
+  // That's 4950 compiles. Assuming each take 1sec to run that ends up at 1.3 hours.
+  //  new FuzzSwapLines,
 )
```

Comment:
I think you should implement two FuzzerSuites:

- One that can run in one minute (and does limited, but deterministic fuzzing).
- And one that aims to take 1 hour to run (also deterministic).

The latter would not be enabled by default.

Perhaps we can have a gradle command `testFuzzer` or similar.

Reply:
I added heuristics to all fuzz tests now. I did not add the testFuzzer command, but it's really easy just to increase the `N` values for each fuzz test suite if we want to hack that together sometime 👍